cmd/attack/kafka.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2022 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package attack
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"go.uber.org/fx"
+
+	"github.com/chaos-mesh/chaosd/cmd/server"
+	"github.com/chaos-mesh/chaosd/pkg/core"
+	"github.com/chaos-mesh/chaosd/pkg/server/chaosd"
+	"github.com/chaos-mesh/chaosd/pkg/utils"
+)
+
+func NewKafkaAttackCommand(uid *string) *cobra.Command {
+	options := core.NewKafkaCommand()
+	dep := fx.Options(
+		server.Module,
+		fx.Provide(func() *core.KafkaCommand {
+			options.UID = *uid
+			return options
+		}),
+	)
+
+	cmd := &cobra.Command{
+		Use:   "kafka <subcommand>",
+		Short: "Kafka attack related commands",
+	}
+
+	cmd.PersistentFlags().StringVarP(&options.Topic, "topic", "T", "", "the topic to attack")
+	cmd.PersistentFlags().IntVarP(&options.Partition, "partition", "", 0, "the partition to attack")
+	cmd.PersistentFlags().BoolVarP(&options.NoSilent, "no-silent", "", false, "print more information")
```

Comment:
Why need this config? Seems that can use log-level to replace.

Reply:
Removed